.github/ISSUE_TEMPLATE/nowa.yaml

```diff
@@ -103,3 +103,8 @@ body:
       placeholder: 000 000 000
     validations:
       required: true
+  - type: markdown
+    attributes:
+      value: |
+        # Następne kroki
+        Po wypełnieniu formularza, Twój wniosek zostanie zweryfikowany przez nasz zespół. W celu weryfikacji poprawności danych skontaktujemy się z Twoją organizacją poprzez oficjalne dane kontaktowe dostępne na stronie internetowej organizacji lub w rejestrze KRS. W przypadku pozytywnej weryfikacji, otrzymasz od nas informację o dalszych krokach w komentarzu do tego zgłoszenia (zakładka Issues).
```

Comment:
Skoro markdown to może od razu by dać link do zakładki `Issues`?

Reply:
Nawet zacząłem dodawać, ale w tym miejscu (formularz) nie chcemy przekierowywać nikogo poza stronę (zaraz niżej ma przycisk submit).